soc/arm/nxp_imx/rt/Kconfig.soc

```diff
@@ -263,6 +291,9 @@ config INIT_USB1_PLL
 config INIT_VIDEO_PLL
 	bool "Initialize Video PLL"
 
+config HAS_NO_ARM_DIV
```

Comment:
Let's make this positive logic instead. Change to `HAS_ARM_DIV`, set default y here (since that is the typical case for most socs in this series), then override the default in `Kconfig.defconfig.mimxrt1010` only

Reply:
Thanks, updated as suggested.